Cargo.toml

```diff
@@ -35,5 +35,10 @@ quicli = { version = "0.4", optional = true }
 structopt = { version = "0.3.17", optional = true }
 term-table = { version = "1.3.0", optional = true }
 exitfailure = { version = "0.5.1", optional =true }
-env_logger = { version = "0.7.1", optional = true }
+env_logger = { version = "0.9", optional = true }
 serde_json = { version = "1.0", optional = true }
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+# DIRTY HACK WARNING: We don't actually use this, but if we don't specify custom, things die because
```

Comment:
Can we rephrase a bit so this is a more neutral explainer as to why this dependency is here?

Reply:
Better?

Reply:
I was hoping to remove the `DIRTY HACK WARNING` comment.

Reply:
Done!